TelCo.ColorCoder/colorManual.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Drawing;
+using System.Text;
+
+namespace ConsoleApp1
+{
+    class colorManual
+    {
+        int pno;
```

Comment:
Resolve the compiler warnings regarding this `pno` data member.

Reply:
yes done!